reference/commands/export-pkg.rst

```diff
@@ -0,0 +1,67 @@
+.. _reference_commands_export-pkg:
+
+conan export-pkg
+================
+
+.. code-block:: bash
+
+    $ conan export-pkg --help
+    usage: conan export-pkg [-h] [-f FORMAT] [-v [V]] [--logger] [-of OUTPUT_FOLDER] [--name NAME] [--version VERSION]
+                            [--user USER] [--channel CHANNEL] [-l LOCKFILE] [--lockfile-partial] [--lockfile-out LOCKFILE_OUT]
+                            [--lockfile-packages] [--lockfile-clean] [-o OPTIONS_HOST] [-o:b OPTIONS_BUILD] [-o:h OPTIONS_HOST]
+                            [-pr PROFILE_HOST] [-pr:b PROFILE_BUILD] [-pr:h PROFILE_HOST] [-s SETTINGS_HOST]
+                            [-s:b SETTINGS_BUILD] [-s:h SETTINGS_HOST] [-c CONF_HOST] [-c:b CONF_BUILD] [-c:h CONF_HOST]
+                            path
+
+    Create a package directly from pre-compiled binaries
+
+    positional arguments:
+    path                  Path to a folder containing a recipe (conanfile.py)
+
+    optional arguments:
+    -h, --help            show this help message and exit
+    -f FORMAT, --format FORMAT
+                            Select the output format: json
+    -v [V]                Level of detail of the output. Valid options from less verbose to more verbose: -vquiet, -verror,
+                            -vwarning, -vnotice, -vstatus, -v or -vverbose, -vv or -vdebug, -vvv or -vtrace
+    --logger              Show the output with log format, with time, type and message.
+    -of OUTPUT_FOLDER, --output-folder OUTPUT_FOLDER
+                            The root output folder for generated and build files
+    --name NAME           Provide a package name if not specified in conanfile
+    --version VERSION     Provide a package version if not specified in conanfile
+    --user USER           Provide a user if not specified in conanfile
+    --channel CHANNEL     Provide a channel if not specified in conanfil
+    -l LOCKFILE, --lockfile LOCKFILE
+                            Path to a lockfile. Use --lockfile="" to avoid automatic use of existing 'conan.lock' file
+    --lockfile-partial    Do not raise an error if some dependency is not found in lockfile
+    --lockfile-out LOCKFILE_OUT
+                            Filename of the updated lockfile
+    --lockfile-packages   Lock package-id and package-revision information
+    --lockfile-clean      remove unused
+    -o OPTIONS_HOST, --options OPTIONS_HOST
+                            Define options values (host machine), e.g.: -o Pkg:with_qt=true
+    -o:b OPTIONS_BUILD, --options:build OPTIONS_BUILD
+                            Define options values (build machine), e.g.: -o:b Pkg:with_qt=true
+    -o:h OPTIONS_HOST, --options:host OPTIONS_HOST
+                            Define options values (host machine), e.g.: -o:h Pkg:with_qt=true
+    -pr PROFILE_HOST, --profile PROFILE_HOST
+                            Apply the specified profile to the host machine
```

Comment:
I think it specifies both host and build at the same time per https://github.com/conan-io/tribe/pull/23